### DISCUSSION TITLE
zls怎么配置

### DISCUSSION THREAD

**Discussion by dengyunsheng250** [zigcc/forum#67] [新手提问]
vscode撇子zls后，显示无法连接到server

**Comment by jiacai2050** [comment]
爱莫能助，我用 Emacs。。

**Comment by mwc** [comment]
比如下载解压到 ~/zig/zls，打开 vscode settings，找到 Zig > Zls: Path 项，设置为 ~/zig/zls

**Comment by jinzhongjia** [comment]
最新的vscode-zig支持自动下载zls的nightly版本，它会先检测你是否有zls，然后报错没有zls，再问你是否要下载zls
